backend/api/views/friendView.py

```diff
@@ -54,8 +54,52 @@ def friends_api(request, id):
             return Response({"detail":"node not ready, no remote request"}, status=403)
     else:
         # remote
-        # only allow get from remote
-        if request.method == 'GET':
+        # only allow get, delete from remote
+        if request.method != 'POST':
```

Comment:
if only get, delete are allowed, `if request.method == 'GET' or request.method == 'DELETE'` is better than negate `POST`

Reply:
view itself disallow other methods, so it's same as the or, and shorter